packages/client/src/testing/fixtures.ts

```diff
@@ -153,7 +153,10 @@ export const archiveMock = {
 export const flagMock = {
   request: {
     query: graphql.FlagDocument,
-    variables: { conversationIDs: [conversation.id], isFlagged: false }
+    variables: {
+      conversationIDs: [conversation.id],
+      isFlagged: conversation.isStarred
```

Comment:
It's a little weird to reference `conversation.isStarred` for the expected variable value, and to use a literal value `true` for the `isStarred` value in the mock response. Either both values should be `true`, or both should be `false`.

Something you can do that might make testing smoother is to make `flagMock` a function instead of a plain value. It could accept an argument for `isStarred` to make it easier to use the mock for both the star and unstar cases.